ci/__main__.py

```diff
@@ -5,23 +5,18 @@
 PYTHON_VERSIONS = ["3.6"]
 
 ADDITIONAL_CORE_DEPS = [
-    "pyface==6.0.0-2",
+    "pyface==6.1.0-2",
     "pygments==2.2.0-1",
-    "pyqt==4.11.4-7",
-    "qt==4.8.7-7",
-    "sip==4.17-4",
-    "traitsui==6.0.0-2",
+    "pyqt==4.12.1-1",
```

Comment:
Be careful of pyqt 4.12 - it has been buggy in the unittests for Pyface an TraitsUI where 4.11 has been comparatively stable.

Reply:
This suggestion might save my day. Checking!

Reply:
It doesn't seem to affect my case. Pushing anyway, because I have different set of test failures locally and in CI.